Source/Core/DolphinQt/Config/Mapping/MappingCommon.cpp

```diff
@@ -102,10 +109,103 @@ void TestOutput(QPushButton* button, OutputReference* reference)
   QApplication::processEvents();
 
   reference->State(1.0);
-  Common::SleepCurrentThread(OUTPUT_TEST_TIME);
+  Common::SleepCurrentThread(OUTPUT_TEST_TIME_MS);
```

Comment:
Not a big deal and not sure what advantages `SleepCurrentThread` has but if we used `std::this_thread::sleep_for(...)` we could use a chrono time, which would make `OUTPUT_TEST_TIME` consistent with the rest of the values in `MappingCommon`

Reply:
I'll change it.